std/compress/lzss/compress.go

```diff
@@ -103,7 +103,7 @@ func (compressor *Compressor) Compress(d []byte) (c []byte, err error) {
 
 	// reset output buffer
 	compressor.buf.Reset()
-	compressor.buf.WriteByte(byte(compressor.level))
+	compressor.buf.Write([]byte{0, byte(compressor.level)}) // 0 -> compressor release version
```

Comment:
can you extract in a `writeMetadata` or `writeHeader`/ corresponding `readMetadata` / `readHeader`?